components/docs-chef-io/content/automate/ha_aws_deploy_steps.md

```diff
@@ -85,54 +85,105 @@ Run the following steps on Bastion Host Machine:
-        - Set `ami_id`, this value depends on your AWS Region and the Operating System Image you want to use.
-        - Please use the [Hardware Requirement Calculator sheet](/calculator/automate_ha_hardware_calculator.xlsx) to get information for which instance type you will need for your load.
-        - Set Instance Type for Chef Automate in `automate_server_instance_type`.
-        - Set Instance Type for Chef Infra Server in `chef_server_instance_type`.
-        - Set Instance Type for OpenSearch in `opensearch_server_instance_type`.
-        - Set Instance Type for Postgresql in `postgresql_server_instance_type`.
-        - Set `automate_lb_certificate_arn` with the arn value of the Certificate created in AWS ACM for DNS entry of `chefautomate.example.com`.
-        - Set `chef_server_lb_certificate_arn` with the arn value of the Certificate created in AWS ACM for DNS entry of `chefinfraserver.example.com`.
-        - Set `automate_ebs_volume_iops`, `automate_ebs_volume_size` based on your load needs.
-        - Set `chef_ebs_volume_iops`, `chef_ebs_volume_size` based on your load needs.
-        - Set `opensearch_ebs_volume_iops`, `opensearch_ebs_volume_size` based on your load needs.
-        - Set `postgresql_ebs_volume_iops`, `postgresql_ebs_volume_size` based on your load needs.
-        - Set `automate_ebs_volume_type`, `chef_ebs_volume_type`, `opensearch_ebs_volume_type`, `postgresql_ebs_volume_type`. Default value is `"gp3"`. Change this based on your needs.
+    - Select `Chef Automate HA` as topology
+    ```bash
+    Choose Topology:
+        Chef Automate HA
+    ```
+    - select deployment type as `AWS`
+    ```bash
+    Choose type of Deployment Type:
+       On-Premise
+     > AWS
+    ```
+    - provide  `ssh user name`, `ssh group name`, `ssh port no`, `ssh key file path`
+    - if you have own certificates for Automate, ChefServer, OpenSearch and Postgresql, then select `yes` and provide relevant certificates. Click [here](/automate/ha_cert_deployment) to know more on adding certificates for services during deployment.
+    ```bash
+        Will you use custom certs for any service like Automate, Chef Infra Server, PostgreSQL, OpenSearch:
+        > no
+          yes
+    ```
+    - provide `AWS profile name` skip this if IAM role configured on bashtion host
+    - filter and select AWS region from list
+    - give AWS VPC ID created in the Prerequisites step. Example: `"vpc12318h"`
+    - to create subnets we have two options as CIDR Block and subnets ids, select  `yes` for CIDR and `no` for subnet ids, and provide subnet ids created in the Prerequisites step. Example `subnet-07e469d218301533`, subnets should be created under same VPC provided above, we need three private subnets, if you want to keep loadbalancer on public IP then we need three public subnets as well, recomended is to use subnet ids.
```

Comment:
This point is suggesting to choose yes for `CIDR` but then its suggesting to provide subnet ids which is wrong,If this point is about using subnets instead of CIDR then it should be mentioned like `No` for CIDR and `Yes` for subnet ids